src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -461,35 +380,8 @@ export default Vue.extend({
         }
       }
     },
-    // check to see if a set of options (college, major, minor) only has placeholder texts (so no options selected)
-    noOptionSelected(options) {
-      let bool = true;
-      options.forEach(option => {
-        if (option.placeholder !== placeholderText) {
-          bool = false;
-        }
-      });
-
-      return bool;
-    },
-    notPlaceholderOptions(options) {
-      const list = [];
-      options.forEach(option => {
-        if (option.placeholder !== placeholderText) {
-          const obj = {
-            acronym: option.acronym,
-            fullName: option.placeholder,
-          };
-
-          list.push(obj);
-        }
-      });
-
-      return list;
-    },
```

Comment:
Just checking but is this dead code?

Reply:
Yes this is dead. The vue template isn't using it